src/jurisdictions/PA/dictionary.txt

```diff
@@ -256,6 +256,7 @@ CandidateContest	Senator in the General Assembly;6th Senatorial District	Pennsyl
 CandidateContest	Senator in the General Assembly;8th Senatorial District	Pennsylvania;General Assembly Senate District 8
 CandidateContest	United States Senator;Statewide	Pennsylvania;US Congress;Senate
 ReportingUnit	Pennsylvania;ADAMS County	ADAMS
+ReportingUnit	Pennsylvania;ADAMS County	ADAMS
```

Comment:
I think this line was likely added for testing purposes, and we probably do not want to add into the repo. Otherwise, this PR looks good to me.